### ISSUE TITLE
In io module, automatically evaluate module_file on given path

### ISSUE THREAD

**Issue by mschubert** [mschubert/ebits#4]
This would make sure that read/write is always relative to the script the call resides in.

Any downsides?

**Comment by mschubert** [comment]
attaching this `base` operator currently also breaks e.g. the `BatchJobs` package

**Comment by klmr** [comment]
Please report a separate bug for this.

**Comment by mschubert** [comment]
Agreed, this was more for a quick observation not to get lost while not having done any tests - I'll still add it as-is, maybe to just declare is invalid after.

**Comment by klmr** [comment]
Do we actually still need this?

The only difference to the  lambda operator is that it allows defaults for optional arguments, and that it can be used even if the user wants to preserve `<-` for assignment. And on the downside it has an _almost, but not quite_ identical syntax with ever so slight inconsistencies, which makes it harder to use.

**Comment by mschubert** [comment]
I'd vote for making the `x ~ y -> x + y` syntax the (only) default and reconsider if we run into serious issues.

If this is acceptable to you you can close this.
